# an empty map will use the currently loaded map
string map_name

string name
---
map_manager/Zone zone
bool success
string message